dev/manifest.txt

```diff
@@ -0,0 +1,22 @@
+app
+bin
+config
+conjur
+db
+doc
+lib
+policy
+public
+vendor
+CHANGELOG.md
+config.ru
+Gemfile
+Gemfile.lock
+LICENSE
+manifest.yml
+NOTICES.txt
+push-image.sh
```

Comment:
I **think** we can omit `push-image.sh`, as it's only used in Jenkinsfile.
Also, should `push-image.sh` be moved to the new `test` directory?

Reply:
I moved it to the `dev` directory, since it looks like it's used more for development purposes than testing